acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
The way the resolver is written now, the lines below are correct on all currently tested platforms, and this test would pass with or without my change on all unsupported platforms, because the code in the resolver _always_ sets the major version equal to the full version. In the case of an "unrecognized" kernel version, it sets them both equal to the full kernel version string. I assumed that we would want to treat Windows 10 the same as we are treating Windows 7 and 8, i.e. return "10" as the full and major version, which is also identical to the way we treat server versions. The full kernel string should _only_ appear in the "kernel" fact.